### ISSUE TITLE
6.4 Implementing a 1D convnet for the IMDB dataset

### ISSUE THREAD

**Issue by yurisolovev** [fchollet/deep-learning-with-python-notebooks#124]
In the Dense layer, a sigmoidal activation function must be added, since we solve the classification problem, right?

**Comment by oren0e** [comment]
Yep, I believe this is a mistake in the book. Until I added the sigmoid activation the loss wasn't changing with the epochs.
